cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/OrderByTest.scala

```diff
@@ -38,11 +38,13 @@ class OrderByTest extends DocumentingTest {
     p("""* <<order-introduction, Introduction>>
         #* <<order-nodes-by-property, Order nodes by property>>
         #* <<order-nodes-by-multiple-properties, Order nodes by multiple properties>>
+        #* <<order-nodes-by-id, Order nodes by id>>
         #* <<order-nodes-in-descending-order, Order nodes in descending order>>
         #* <<order-null, Ordering `null`>>""".stripMargin('#'))
     section("Introduction", "order-introduction") {
-      p("""Note that you cannot sort on nodes or relationships, just on properties on these.
-          #`ORDER BY` relies on comparisons to sort the output, see <<cypher-ordering, Ordering and comparison of values>>.""".stripMargin('#'))
+      p("""`ORDER BY` relies on comparisons to sort the output, see <<cypher-ordering, Ordering and comparison of values>>.
+          #You can sort on either node/relationship properties, or the node/relationship ids.
```

Comment:
Perhaps we should change the phrasing a little to make it clear that you can sort on almost anything, not only properties and ids. Most expressions can be used to sort. Like `MATCH (n) RETURN n ORDER BY labels(n);`, `UNWIND ['ac', 'ba', 'cb'] AS a RETURN a ORDER BY right(a, 1);`, etc.